extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,522 @@
+
+This extension adds the `mimeType` property to identify the type of content. This is more reliable than a file extension for determining the file type.
+
+To use glTF models (`.gltf` files) or binary glTF models (`.glb` files) as content, the `3DTILES_content_gltf` extension must be listed as a required extension in the tileset.
+
+## Subtrees
+
+>**Subtrees** are fixed-depth and fixed-branching factor sections of the tileset tree used for breaking tilesets into manageable pieces.
+
+Since tilesets grow exponentially with depth, storing information about every tile in a single file is not always feasible or desirable. Even if RAM is not a direct limitation, streaming large files over the network can make loading times slower. To account for this, subtrees partition the tileset structure into pieces of bounded size.
+
+![exact cover](figures/union-of-subtrees.jpg)
+
+Each subtree is a tree-shaped container for tiles. A subtree has a fixed number of levels defined by the `subtreeLevels` property. This describes the number of distinct levels in the tree. The branching factor is also fixed due to the tiling scheme. For quadtrees, the branching factor is `4`, while octrees have a branching factor of `8`. Taken together, a subtree has exactly enough nodes to store a full quadtree or full octree with a limited depth. However, each node may or may not contain a tile, as a tileset only stores the tiles that are necessary.
+
+![subtree anatomy](figures/subtree-anatomy.jpg)
+
+OUTLINE:
+- refering to nodes is confusing. review this!
+
+## Availability
+
+**Availability** is boolean data about which tiles, contents, or subtrees exist in a tileset. Availability serves two purposes:
+
+1. It provides an efficient method for checking what files are present
+2. Including ths information prevents extraneous HTTP requests that would result in 404 errors.
+
+Availablity takes the form of a bitstream with one bit per node in consideration. A 1 indicates that a tile/content/subtree is available at this node. Meanwhile, a 0 indicates that no data is available.
+
+When every node is available or every node is unavailable, all the bits of the bitstream will be identical. That is, either all 1s or all 0s. Instead of storing a full bit stream, the `constant` property can be used instead. For example, `constant: 0` indicates that all bits are 0 and no bitstream must be stored.
+
+Availability data is scoped to a subtree. This ensures that the size of each bitstream is bounded to a reasonable size.
+
+### Tile Availability
+
+**Tile availability** is a bitstream that determines which tiles exist within a subtree. There is one bit for each subtree node. A 1 indicates that a tile is available, while a 0 indicates that a tile is unavailable.
+
+![Tile Availability](figures/tile-availability.jpg)
```

Comment:
This diagram may potentially mislead someone into believing that binary trees are supported. We should use quadtrees.

Reply:
Yeah, my only hesitation is that a quadtree diagram could get quite a bit wider (since we need to show the entire buffer). 

As a group we decided to hold off for now and find a better solution later.